app/ui/generalpage.ui

```diff
@@ -458,14 +458,17 @@
               <height>16777215</height>
              </size>
             </property>
+            <property name="suffix">
+             <string>MB</string>
```

Comment:
Technically this, as well as in the other places where MB is used, should be MiB.

Reply:
While that's technically correct, Not many are familiar with this term, neither windows or mac give you mib, I believe Ubuntu does but frankly it's not something I've ever noticed.

Reply:
Didn't see MiB very often to be honest. In many places, MB refers to the same meaning of MiB.